### PR TITLE
Feature/login hookup

### PR DESCRIPTION
**Pull request by eliabejr**

### CODE REVIEW

File: src/components/FullLoader.re [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  let logo = style([width(Pt(200.))]);
+};
+
+let component = ReasonReact.statelessComponent("FullLoader");
+
+let make = (~mode, ~message=?, _children) => {
+  ...component,
+  render: _self =>
+    <Shimmer style=Styles.wrapper>
+      <View style=Style.(style([alignItems(Center)]))>
```
Comment (medson10) [inline src/components/FullLoader.re]: inline style

File: src/scenes/InAppNavigator.re [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+let make = (~navigation as mainNavigation, _children) => {
+  ...component,
+  render: _self =>
+    <StackNavigator.Screen
+      headerTitle="InAppNavigator" navigation=mainNavigation>
+      ...{() =>
+        <SafeAreaView
+          style=Style.(
+            style([
+              flex(1.),
```
Comment (medson10) [inline src/scenes/InAppNavigator.re:31]: inline style

File: src/scenes/InAppNavigator.re [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            ])
+          )>
+          <RNTabView
+            tabBarPosition=`bottom
+            renderTabBar={rest =>
+              <RNTabView.TabBar
+                rest
+                indicatorStyle=Style.(
+                  style([
+                    backgroundColor(String(AppConfig.theme.primary)),
```
Comment (medson10) [inline src/scenes/InAppNavigator.re]: inline style, same below

File: src/scenes/Login.re [hunk 4]
```diff
@@ -13,22 +14,129 @@ module Styles = {
...
+                  keyboardType=`emailAddress
+                  value={form.values.email}
+                  style=Style.(style([marginBottom(Pt(10.))]))
+                />
+                <AppButton
+                  onPress={_ => handleSubmit()}
+                  text="Login"
+                  disabled={
+                    Belt.Option.isSome(getErrorForField(`email))
+                    || form.values.email == ""
```
Comment (medson10) [inline src/scenes/Login.re]: better use string length instead of == ""

File: src/Epitath.re [hunk 5]
```diff
@@ -0,0 +1 @@
+let let_ = (children, render) => children(render);
```
Comment (fakenickels) [inline src/Epitath.re]: remove and use just from bs-epitath!

File: src/types/TabView.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+  render() {
+    const sceneMapConfig = this.props.scenes.reduce(
+      (config, scene) => ({ ...config, [scene.key]: scene.render }),
+      {},
+    )
+    return (
+      <TabView
+        navigationState={this.state}
+        renderScene={SceneMap(sceneMapConfig)}
```
Comment (medson10) [inline src/types/TabView.js:22]: why not with bindings?

**Review by fakenickels — APPROVED** [review]

File: src/types/TabView.js (hunk 6 above)
Comment (eliabejr) [reply src/types/TabView.js:22]: really don't know 🤔 got it from last

Comment (eliabejr) [reply src/types/TabView.js:22]: just put here the files needed to inside app navigation

Comment (medson10) [reply src/types/TabView.js:22]: hmm okay, can change later for now it's okay

**Review by medson10 — APPROVED** [review]

**Pull request merged**